helpers/get.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const _ = require('lodash');
+
+/**
+ * Based on https://github.com/helpers/handlebars-helpers/blob/0.9.0/lib/object.js#L128-L134
+ * 
+ * Get a value from the given context object. Property paths (`a.b.c`) may be used
+ * to get nested properties.
+ */
+const factory = () => {
+    return function (path, context) {
+        let options = arguments[arguments.length - 1];
+
+        // use an empty context if none was given
+        if (arguments.length < 2) {
+            context = {};
+        }
+
+        // property name/path must be a string and a direct property of the object
+        if (typeof path !== 'string' || !_.has(context, path)) {
+            return;
+        }
+
+        let value = _.get(context, path);
```

Comment:
If it possible not to bring dependency on lodash, it would be ideal. We have a ticket in backlog to remove that dependency.

Reply:
Done.